nondex-instrumentation/src/test/java/edu/illinois/nondex/instr/InstrumenterTest.java

```diff
@@ -28,33 +28,39 @@ a copy of this software and associated documentation files (the
 
 package edu.illinois.nondex.instr;
 
-import static org.hamcrest.core.Is.is;
+
+import static org.hamcrest.CoreMatchers.containsString;
 import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
 import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.LinkOption;
+import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.security.NoSuchAlgorithmException;
 
 import org.junit.Test;
 
 public class InstrumenterTest {
     @Test
-    public void emptyZipTest() {
-        try {
-            Instrumenter.instrument(Paths.get("resources", "empty.jar"), Paths.get("resources", "emptyOut.jar"));
-            fail("Expected an IOException to be thrown");
-        } catch (IOException exc) {
-            assertThat(exc.getMessage(), is("zip file is empty"));
-        }
+    public void emptyZipTest() throws NoSuchAlgorithmException, IOException {
+        Path outPath = Paths.get("resources", "emptyOut.jar");
+        Instrumenter.instrument(Paths.get("resources", "empty.jar").toString(),
+                                outPath.toString());
+        assertTrue(Files.exists(outPath, LinkOption.NOFOLLOW_LINKS));
```

Comment:
After running this test, will the emptyOut.jar be created? Should we remove it after the run?